modules/post/osx/gather/enum_osx.rb

```diff
@@ -176,35 +153,18 @@ def enum_conf(log_folder)
     # Enumerate first using System Profiler
     profile_datatypes.each do |name, profile_datatypes|
       print_status("\tEnumerating #{name}")
-      # Run commands according to the session type
-        if session.type =~ /meterpreter/
-          returned_data = cmd_exec('system_profiler', profile_datatypes)
-          # Save data lo log folder
-          file_local_write(log_folder+"//#{name}.txt",returned_data)
-        elsif session.type =~ /shell/
-          begin
-            returned_data = cmd_exec("/usr/sbin/system_profiler #{profile_datatypes}")
-            # Save data lo log folder
-            file_local_write(log_folder+"//#{name}.txt",returned_data)
-          rescue
-          end
-        end
+      returned_data = cmd_exec("/usr/sbin/system_profiler #{profile_datatypes}")
```

Comment:
The module exclusively uses `cmd_exec(cmd, arg)` format with two arguments, where as in this instance it's using `cmd_exec(cmd arg)` with one argument.

Not an issue, just inconsistent.